src/Workleap.DomainEventPropagation.Analyzers/Internals/RuleIdentifiers.cs

```diff
@@ -7,4 +7,6 @@ internal static class RuleIdentifiers
     // DO NOT change the identifier of existing rules.
     // Projects can customize the severity level of analysis rules using a .editorconfig file.
     public const string UseDomainEventAttribute = "WLDEP01";
+
+    public const string UseUniqueNameForAttributeValue = "WLDEP02";
```

Comment:
This class has a MediatR copy-pasted namespace.
Same thing for RuleCategories.
After removing them, make sure the old MediatR unused usings are removed too.